src/test/ij.c

```diff
@@ -4148,11 +4220,11 @@ main( hypre_int argc,
       HYPRE_ParaSailsSetFilter(pcg_precond, 0.);
       HYPRE_ParaSailsSetLogging(pcg_precond, ioutdat);
 
-      HYPRE_IJMatrixGetObject( ij_A, &object);
-      parcsr_mat = (HYPRE_ParCSRMatrix) object;
+      HYPRE_IJMatrixGetObject(ij_A, &object);
+      parcsr_A = (HYPRE_ParCSRMatrix) object;
 
-      HYPRE_ParaSailsSetup(pcg_precond, parcsr_mat, NULL, NULL);
-      HYPRE_ParaSailsBuildIJMatrix(pcg_precond, &ij_M);
+      HYPRE_ParaSailsSetup(pcg_precond, parcsr_A, NULL, NULL);
```

Comment:
why it is not set up with `parcsr_M` here? It seems that `parcsr_M` is used in other setups

Reply:
I missed that! Thanks for catching!

Reply:
so basically, from now on we should set up a preconditioner with `M` in the `ij` driver? If so, we should give all a heads-up

Reply:
Yes, good idea! I'll mention that on Teams